tests/app/views/test_suppliers.py

```diff
@@ -1320,6 +1320,136 @@ def test_get_supplier_framework_info(self, open_g8_framework):
             'uploaderUserId': 20
         }
 
+    def test_get_supplier_framework_returns_framework_agreement(self, live_example_framework):
```

Comment:
Same as above, currently working under the assumption of only one framework agreement per supplier framework.